### DISCUSSION TITLE
How to make the Progress component to a circle?

### DISCUSSION THREAD

**Discussion by Ding-Fan** [radix-ui/primitives#1827] [Help]
Currently the [progress component](https://www.radix-ui.com/docs/primitives/components/progress) is a bar. I want to make a progress circle. One of the popular way is draw a `<circle>` in `<svg>` . It seems that [Progress.Indicator was a Primitive.div](https://github.com/radix-ui/primitives/blob/main/packages/react/progress/src/Progress.tsx#L99) .

Is there any way to do it?

**Comment by benoitgrelard** [comment]
You could do it with `asChild` on the root and on the indicator to make them an svg and circle respectively.
Note that we haven't tested accessibility with an svg though.
